client/layout/guided-tours/positioning.ts

```diff
@@ -200,7 +200,7 @@ function scrollIntoView( target: Element, scrollContainer: Element | null ) {
 
 	if ( isMobile() ) {
 		clientHeight = document.documentElement.clientHeight;
-	} else if ( window === scrollContainer ) {
+	} else if ( ( ( window as unknown ) as Element ) === scrollContainer ) {
```

Comment:
are you familiar with ambient global types?
we might benefit from creating a global interface for `Window` that incorporates the things we're going to be referencing and using.

Reply:
alternatively here I guess `scrollContainer` is a different type than `window` and it could be nice as well to de-cast both sides of the equality

```ts
( window as unknown === scrollContainer as unknown )
```

this is truly a kind of reference-equality check. your code is totally fine as written!

Reply:
I like that better; thanks! If we're ignoring the type of `window` here we might as well ignore both.